code/wdb/datastripe.q

```diff
@@ -22,14 +22,12 @@ modaccess:{[accesstab]};
     // on first save down we need to replace the null valued start time in the access table
     // using the first value in the saved data
     starttimes:.ds.getstarttime each t;
-    .wdb.access:update start:starttimes^start, end:?[(nextp>starttimes)&(starttimes<>0Np);nextp;0Np], stptime:data[][`time] from .wdb.access;
+    .wdb.access:update start:starttimes^start, end:?[(nextp>starttimes)&(starttimes<>0Np);nextp;0Np], stptime:data[][`p] from .wdb.access;
```

Comment:
so was referencing `time` before a bug? I can't see from context or PR description etc. why this is changed

Reply:
Updating the `checkends[]` function in `segmentedtickerplant/stplog.q` resolves the bug with referencing `time`; the change here to reference `p` is to populate `stptime` with the offset UTC time `.z.p+.eodtime.dailyadj`, as opposed to the local time `.z.P`.

Reply:
OK - so why are we making that change to use offset UTC time instead? And if someone looks at this piece of code in 6 months, how are they going to know what `p` means here? Is that documented somewhere?

Reply:
Understood - but what is the reason for that suggestion? (my point here is basically that you should generally have at least some understanding of the reason/justification for the change you are making)

Regarding `p` - the fact that it's used elsewhere may not be that helpful to someone looking at this line of code. Ideally I shouldn't have to go digging through a bunch of other code to understand what a variable/name here means. It's not apparent looking at this piece of code where the `data` object might be coming from, or what it contains etc. - even adding a comment here to explain the structure of `data` & what `p` means would be very helpful IMO

Reply:
Had a chat with Callum Reid, and updated to use the `time` field of `.stplg.endofperioddata[]`, i.e. the local time `.z.P`.